estate/models/model_real_estate_property.xml

```diff
@@ -0,0 +1,130 @@
+        <field name="name">x_expected_price</field>
+        <field name="field_description">Expected Price</field>
+        <field name="ttype">float</field>
+        <field name="required">True</field>
+    </record>
+
+    <record id="field_real_estate_property_bedrooms" model="ir.model.fields">
+        <field name="model_id" ref="estate.model_real_estate_property" />
+        <field name="name">x_bedrooms</field>
+        <field name="field_description">Bedrooms</field>
+        <field name="ttype">integer</field>
+    </record>
+
+    <record id="field_real_estate_property_living_area" model="ir.model.fields">
+        <field name="model_id" ref="estate.model_real_estate_property" />
+        <field name="name">x_living_area</field>
+        <field name="field_description">Living Area</field>
+        <field name="ttype">integer</field>
+    </record>
+
+    <record id="field_real_estate_property_facades" model="ir.model.fields">
+        <field name="model_id" ref="estate.model_real_estate_property" />
+        <field name="name">x_facades</field>
+        <field name="field_description">Facades</field>
+        <field name="ttype">integer</field>
+    </record>
+
+    <record id="field_real_estate_property_garage" model="ir.model.fields">
+        <field name="model_id" ref="estate.model_real_estate_property" />
+        <field name="name">x_garage</field>
+        <field name="field_description">Garage</field>
+        <field name="ttype">boolean</field>
+    </record>
+
+    <record id="field_real_estate_property_garden" model="ir.model.fields">
+        <field name="model_id" ref="estate.model_real_estate_property" />
+        <field name="name">x_garden</field>
+        <field name="field_description">Garden</field>
+        <field name="ttype">boolean</field>
+    </record>
+
+    <record id="field_real_estate_property_garden_area" model="ir.model.fields">
+        <field name="model_id" ref="estate.model_real_estate_property" />
+        <field name="name">x_garden_area</field>
+        <field name="field_description">Garden Area</field>
+        <field name="ttype">integer</field>
+    </record>
+
+    <record id="field_real_estate_property_garden_orientation" model="ir.model.fields">
+        <field name="model_id" ref="estate.model_real_estate_property" />
+        <field name="name">x_garden_orientation</field>
+        <field name="field_description">Garden Orientation</field>
+        <field name="ttype">selection</field>
+    </record>
+
+    <record id="selection_real_estate_property_garden_orientation" model="ir.model.fields.selection">
+        <field name="field_id" ref="estate.field_real_estate_property_garden_orientation"/>
+        <field name="name">North</field>
+        <field name="value">north</field>
+        <field name="sequence">1</field>
+    </record>
+
+    <record id="selection_real_estate_property_garden_orientation" model="ir.model.fields.selection">
```

Comment:
same ID as previous one, will edit it instead of creating a new one...
```suggestion
    <record id="selection_real_estate_property_garden_orientation_south" model="ir.model.fields.selection">
```